cdap-docs/_common/common-build.sh

```diff
@@ -131,8 +131,11 @@ function usage() {
 
 function clean() {
   cd $SCRIPT_PATH
-  rm -rf $SCRIPT_PATH/$BUILD
-  mkdir $SCRIPT_PATH/$BUILD
+  if [ -d "${SCRIPT_PATH}/${BUILD}" ]
+  then
+    rm -rf $SCRIPT_PATH/$BUILD
+  fi
+  mkdir -p $SCRIPT_PATH/$BUILD
```

Comment:
I could be wrong, but this is looking more verbose for not much benefit. There's a number of places where one line is replaced with more? Especially cases that then end up with a path that needs to be the same in multiple places.